test/e2e/manifest.yaml

```diff
@@ -0,0 +1,11 @@
+# Test manifest for https://github.com/kubernetes/kubernetes/tree/master/test/e2e/storage/external
+ShortName: manila
+StorageClass:
+  FromName: true
+SnapshotClass:
+  FromName: true
+DriverInfo:
+  Name: manila.csi.openstack.org
+  SupportedSizeRange:
+    Min: 1Gi
+    Max: 16Ti
```

Comment:
I'm going minimal for now, as I'm not sure what are the actual capabilities, but if anyone knows more about them, I'll be happy to add it here.